client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.scss

```diff
@@ -34,3 +36,102 @@
     max-width: 700px;
   }
 }
+
+.data-files-btn {
+  background-color: var(--global-color-accent--normal);
+  border-color: var(--global-color-accent--normal);
+  border-radius: 0;
+  font-family: 'Roboto, sans-serif';
+}
+
+.data-files-btn-cancel {
+  border-radius: 0;
+}
+
+#path-button-wrapper {
+  padding-left: var(--horizontal-buffer);
+}
+#data-files-path {
+  font-size: 12px;
+  padding: 5px 10px;
+  border-color: #707070;
+}
```

Comment:
Why use an ID? It seems to cause unnecessary specificity.

Consider reading [UI - CSS Style Guide - Specificity: IDs in CSS](https://confluence.tacc.utexas.edu/display/~wbomar/UI+-+CSS+Style+Guide+-+Specificity#UICSSStyleGuideSpecificity-IDsinCSS).

Reply:
Removed ID

Reply:
👍 Confirmed (d566301).